forgerock-auth/src/main/java/org/forgerock/android/auth/AccessToken.kt

```diff
@@ -25,15 +34,21 @@ import java.util.StringTokenizer
  * @property scope The scope of the access token which defines the resources that the access token can access.
  * @property sessionToken The session token associated with the access token.
  */
-class AccessToken(override val value: String,
-                  val expiresIn: Long,
-                  var expiration: Date?,
-                  val refreshToken: String?,
-                  val idToken: String?,
-                  val tokenType: String?,
-                  val scope: Scope?,
-                  val sessionToken: SSOToken?) : Token(
-    value), Serializable {
+@Serializable
```

Comment:
Use Kotlin Serialization to make the AccessToken serializable to and from JSON, rather than converting it programmatically.